src/modules/ProductLicense.rb

```diff
@@ -1144,12 +1144,24 @@ def AskLicenseAgreement(src_id, dir, patterns, action, enable_back, base_product
       end
 
       licenses_ref = arg_ref(licenses)
-      DisplayLicenseDialog(
+
+      label = Pkg::SourceGeneralData(src_id)["name"]
+      title = _("License Agreement")
+
+      if !label.empty?
+        title = _("%s License Agreement") % label
+      end
+
+      eula_url = Pkg::SourceGeneralData(src_id)["url"]
+      Yast::ProductLicense.license_file_print = eula_url
```

Comment:
The problem is that there is no http://download.opensuse.org/distribution/13.2/repo/oss/license.txt file.

There is http://download.opensuse.org/distribution/13.2/repo/oss/license.tar.gz file which contains license + all translations in one tarball.